model_analyzer/analyzer.py

```diff
@@ -149,3 +137,31 @@ def _process_top_results(self):
 
             # Send top_n measurements to report manager
             self._report_manager.add_result(result=result)
+
+    def _save_top_models(self):
```

Comment:
Is this in addition to the output_model_repository that we had?

Reply:
Yes, this saves the best model configs (across all models) in the export path.